include/rocksdb/utilities/env_librados.h

```diff
@@ -0,0 +1,189 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+#ifndef ROCKSDB_UTILITIES_ENV_LIBRADOS_H
+#define ROCKSDB_UTILITIES_ENV_LIBRADOS_H
+
+#include <memory>
+#include <string>
+
+#include "rocksdb/status.h"
+#include "rocksdb/utilities/env_mirror.h"
+
+#include <rados/librados.hpp>
+
+// #include "include/assert.h"
+
+class LibradosWritableFile;
+/*
+  Definition of global variables
+*/
+const std::string DEFAULT_POOL_KEY = "DEFAULT_POOL";
+
+// User defined Options
+struct EnvLibradosOptions {
+  uint64_t buffer_max_size;                             // max buffer size for WritableFile handler object
+  std::map<std::string, std::string> prefix_pool_map;   // custimized object pool for specific prefix
```

Comment:
I think we can simplify this to wal_pool and db_pool.

We *could* add a third pool later for tiering to slower storage, but I don't think it's worth it yet.